Adjustments Environment Lighting

ValGrimm-U3D opened the pull request:
HDRP-479 added a help box to the Environment tab of the Lighting window in HDRP.  I have added the text from that help box to the documentation, and updated the screenshot of the menu (which was missing the volumetric clouds option).

# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [n/a ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [n/a] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [n/a] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [n/a] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
Updating the Environment Lighting page with the changes to the Environment tab of the Lighting Window in HDRP.

---
### Testing status
Describe what manual/automated tests were performed for this PR
Not applicable.

---
### Comments to reviewers
Notes for the reviewers you have assigned.
I added one sentence to the page which was approved in https://github.com/Unity-Technologies/Graphics/pull/6059, and updated the screenshot.

ValGrimm-U3D commented on com.unity.render-pipelines.high-definition/Documentation~/Environment-Lighting.md:
```diff
@@ -44,7 +44,7 @@ The **Environment (HDRP)** section is at the top and has two settings that you c
 | **Profile**             | A [Volume Profile](Volume-Profile.md) for the sky. This Volume Profile must include at least one Sky Volume override. |
 | **Static Lighting Sky** | The sky to use for the indirect ambient light simulation. The drop-down only contains sky types that the **Profile** includes. For example, if the **Profile** includes a **Gradient Sky** Volume override, you can select **Gradient Sky** from this drop-down.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. |
-| **Static Lighting Clouds** | The clouds to use for the indirect ambient light simulation. The drop-down only contains cloud types that the **Profile** includes.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. |
+| **Static Lighting Volumetric Clouds** | The clouds to use for the indirect ambient light simulation. The drop-down only contains cloud types that the **Profile** includes.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. The interaction between the Scene Volume sky type and the Scene's dominant directional light may affect the visual characteristics of ...
```
Ok, thank you.
Although this makes me realize---there doesn't seem to be a description of Volumetric Clouds in the doc, just Static Lighting Clouds. I've added Julien to this PR  as well because he did the original PR that prompted me to update the doc, I should have done that to start but forgot and GitHub suggested you.

So, although the screenshot is now up to date, I'm not sure where to add the extra information about the impact of the dominant directional light.

ValGrimm-U3D commented:
@JulienIgnace-Unity Hi Julien, It seems that the option Static Lighting Volumetric Clouds isn't documented in the manual. What shall I write there?

@adrien-de-tocqueville I'm use the info from Julien's to add a description of the new Static Lighting Volumetric Clouds option to the manual, along with the sentence about the effect of directional light which belongs in that description.

adrien-de-tocqueville commented:
Yep, i restored the line you removed and improved the description of the option, i'll let julien review it

ValGrimm-U3D commented:
> Yep, i restored the line you removed and improved the description of the option, i'll let julien review it

I am a bit confused. I did not remove a line; I changed a name of an option and added a line to it from Julien's PR because it looked like an option was undocumented. I'm going to go ahead and close this PR and confer with Julien about what to do about the new option. Thanks!

adrien-de-tocqueville commented:
In the original version the line **static lighting clouds** was removed
![image](https://user-images.githubusercontent.com/61282776/152547541-8ce4ad83-4e24-4162-8569-981f1f737060.png)

The option **static lighting volumetric clouds** is indeed missing so the PR should be merged.
Addin @AnisB as he is responsible of volumetric clouds

anisunity commented:
@ValGrimm-U3D Please include me in volumetric clouds related changes, thanks!
